tests/cli/Cest2Test.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/**
+ * Class CestTest
+ */
+class Cest2Test extends \Codeception\Test\Unit
+{
+    public static function tearDownAfterClass(): void
+    {
+        static $counter = 0;
+        $counter++;
+        if ($counter > 1) {
+            debug_print_backtrace(DEBUG_BACKTRACE_IGNORE_ARGS);
+            die('2 times cli');
+        }
+        // die('1 time cli');
+        parent::tearDownAfterClass(); // TODO: Change the autogenerated stub
+    }
+
+    /**
+     * @group core
+     */
```

Comment:
`#[Group('core')]` ?

Reply:
This file was committed by accident, I used it for testing Before/After execution few weeks ago.